assets/component-facets.css

```diff
@@ -698,6 +697,7 @@ details.menu-opening .mobile-facets__close svg {
   flex-grow: 1;
   display: flex;
   flex-direction: column;
+  overflow-y: auto;
 }
 
```

Comment:
The product modal is behaving differently than the other modals and increasing in size when in focus. Is that expected? https://screenshot.click/01-08-r1phw-g5occ.mp4 

cc @melissaperreault

Reply:
You're seeing the focus outline "padding" as it were. Box-shadows don't allow gradients so when we try to impose the focus outline (via box-shadow) on a gradient background, the "invisible" background-colored padding is visible.